typedspark/_schema/get_schema_definition.py

```diff
@@ -35,16 +35,53 @@ def get_schema_definition_as_string(
     return imports + schema_string
 
 
-def _get_comment(schema: Type[Schema], col_name: str) -> str:
-    """Return the comment of a given column."""
-    if (
-        hasattr(schema.__annotations__[col_name], "__metadata__")
-        and schema.__annotations__[col_name].__metadata__ is not None
-    ):
-        comment = schema.__annotations__[col_name].__metadata__[0]
+def _create_docstring(schema: Type[Schema]) -> str:
```

Comment:
Can we order the functions such that I can read the doc from top to bottom? So:

get_schema_definition_as_string()
_build_schema_definition_string()
_create_docstring()
_add_lines_with_typehint()
_extract_comment()
_create_typehint_comment()
_add_subschemas()
_replace_literals()

Reply:
That way, a new reader doesn't have to scroll so much. It will also become easier later if we want to split this file into multiple files (which we'll want to do, as it's >100 lines, but that's for a later PR ;).